modules/020-deckhouse/docs/README.md

```diff
@@ -24,3 +24,30 @@ In Deckhouse, this module sets up:
 - **Service for validating Custom Resources**.
 
   The validation service prevents creating Custom Resources with invalid values or adding such values to the existing Custom Resources. Note that it only tracks Custom Resources managed by Deckhouse modules.
+
+
+### DeckhouseReleases
```

Comment:
I wouldn't say I like the name of the CRD as a title for the doc sections. Does it feel like it would be better to describe what the following section is about?

Reply:
Changed